### PR TITLE
SSL cert verification support for policy assignment and custom attributes modules

### PR DESCRIPTION
**Pull request by dkorn**
No new functionality, just adding the SSL cert verifications to modules of which it was missing (were added in parallel to the SSL feature)

### CODE REVIEW

**Comment by dkorn** [comment]
@enoodle @cben small change, please review

File: library/manageiq_policy_assignment.py [hunk 1]
```diff
@@ -65,6 +76,7 @@
     miq_url: 'http://localhost:3000'
     miq_username: 'admin'
     miq_password: '******'
+    verify_ssl: False
```
Comment (cben) [inline library/manageiq_policy_assignment.py]: this example uses http URL, so this parameter is meaningless?

File: library/manageiq_custom_attributes.py [hunk 2]
```diff
@@ -65,26 +76,29 @@
         value: "value 1"
       - name: "ca2"
         value: "value 2"
+    verify_ssl: False
```
Comment (cben) [inline library/manageiq_custom_attributes.py]: again, meaningless with http: miq_url

**Comment by cben** [comment]
LGTM.  I'd perhaps use true in examples to encourage good practices.  (true is redundant as it's the default but I think we give many redundant fields in examples?)

File: library/manageiq_policy_assignment.py (hunk 1 above)
Comment (dkorn) [reply library/manageiq_policy_assignment.py]: right, thanks :+1:

File: library/manageiq_custom_attributes.py (hunk 2 above)
Comment (dkorn) [reply library/manageiq_custom_attributes.py]: right, thanks :+1:

**Pull request merged**